lib/medic/support/runtime_version_manager.ex

```diff
@@ -12,7 +12,7 @@ defmodule Medic.Support.RuntimeVersionManager do
             {output, 0} ->
               output
               |> String.split(" ", trim: true)
-              |> Enum.at(1)
+              |> Enum.at(4)
```

Comment:
I checked `plugins/0` locally and it seems unaffected by the changes to the asdf cli.